### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by manuelalferez** [pmascaraque/iycdangbo#23] [General]
![](https://ik.imagekit.io/300/0IyCDangboResources/BANNER1_RqeUUdexv.jpg?updatedAt=1634987720062)
https://ik.imagekit.io/300/0IyCDangboResources/BANNER1_RqeUUdexv.jpg?updatedAt=1634987720062

**Comment by pmascaraque** [comment]
![](https://ik.imagekit.io/300/0IyCDangboResources/8391afb6-83f8-46b3-813b-9a738afba587_1pwhyWX5xW.jpg?updatedAt=1636114732319)
https://ik.imagekit.io/300/0IyCDangboResources/8391afb6-83f8-46b3-813b-9a738afba587_1pwhyWX5xW.jpg?updatedAt=1636114732319
![](https://ik.imagekit.io/300/0IyCDangboResources/47e7024a-1e7c-4c2e-81de-427331025e92_D0WJrUDXP.jpg?updatedAt=1636114732138)
https://ik.imagekit.io/300/0IyCDangboResources/47e7024a-1e7c-4c2e-81de-427331025e92_D0WJrUDXP.jpg?updatedAt=1636114732138

**Comment by manuelalferez** [comment]
![](https://ik.imagekit.io/300/0IyCDangboResources/Capture_i2_Bbi3N2i.PNG?updatedAt=1636125144304)
https://ik.imagekit.io/300/0IyCDangboResources/Capture_i2_Bbi3N2i.PNG?updatedAt=1636125144304

**Comment by pmascaraque** [comment]
![](https://ik.imagekit.io/300/0IyCDangboResources/Cirujanos_JS-39_pKLh_-17J.jpg?updatedAt=1636127145202)
https://ik.imagekit.io/300/0IyCDangboResources/Cirujanos_JS-39_pKLh_-17J.jpg?updatedAt=1636127145202
